pandas/compat/__init__.py

```diff
@@ -26,14 +33,11 @@
     pa_version_under9p0,
 )
 
-if TYPE_CHECKING:
-    import lzma
-
-PY39 = sys.version_info >= (3, 9)
-PY310 = sys.version_info >= (3, 10)
-PY311 = sys.version_info >= (3, 11)
-PYPY = platform.python_implementation() == "PyPy"
-IS64 = sys.maxsize > 2**32
+IS64 = IS64
```

Comment:
I think if you define these in `__all__` as described in https://github.com/pandas-dev/pandas/pull/49068#discussion_r1000791860 this redundant assignment can be avoided

Reply:
Oh the assignment here is to appease the linter while explicitly `import`ing these. Please see these threads for context:

* https://github.com/pandas-dev/pandas/pull/49068#discussion_r1000051268
* https://github.com/pandas-dev/pandas/pull/49068#discussion_r1000073840

That said, happy to add to `__all__` if we want that